controllers/main.controllers.js

```diff
@@ -41,6 +41,18 @@ exports.deleteProject = async (req, res, next) => {
 // @route   DELETE api/client/:id
 exports.deleteClient = async (req, res, next) => {
   // Delete client with req.params.id
+  try {
+    const { id: _id } = req.params;
+    const docs = await Client.findByIdAndDelete(_id);
+    // console.log(docs);
+    if(docs != null) res.status(200).json(docs);
+    else {
+      res.status(200).json("No client found");
+    }
+  } catch (err) {
+    // console.log(err);
+    res.status(400).json(err);
+  }
 };
```

Comment:
remove log comments rest of code is fine
also format response like this
`{
success: true,
data:{
//data goes here}
}`